packages/core/src/services/zoom.ts

```diff
@@ -64,4 +100,32 @@ export class Zoom extends Service {
 			extent(zoomBarData, (d: any) => d[domainIdentifier])
 		);
 	}
+
+	// filter out data not inside zoom domain
+	// to get better range value for axis label
+	filterDataForRangeAxis(displayData: object[], stacked = false) {
+		const zoomDomain = this.model.get("zoomDomain");
+		const isUpdateRangeAxis = Tools.getProperty(
```

Comment:
```suggestion
		const shouldUpdateRangeAxis = Tools.getProperty(
```

Reply:
Updated in https://github.com/carbon-design-system/carbon-charts/pull/766/commits/9688034ff3f124008639b3de35559c84dee307c4.